core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -572,6 +572,17 @@ public static enum ProtocolVersion {
             this.bitcoinProtocol = bitcoinProtocol;
         }
 
+        /**
+         * @return protocol version as an integer value
+         */
+        public int intValue() {
+            return bitcoinProtocol;
+        }
+
+        /**
+         * @deprecated Use {@link #intValue()}
+         */
+        @Deprecated
         public int getBitcoinProtocolVersion() {
             return bitcoinProtocol;
         }
```

Comment:
Again, the old method could get a `deprecatedMembers()` test.

Reply:
Fixed and force-pushed.